### PR TITLE
Change list to set in sc_module and sc_server

### PR DESCRIPTION
**Pull request by FalseR20**

### CODE REVIEW

File: src/sc_kpm/sc_server.py [hunk 1]
```diff
@@ -54,7 +58,7 @@ def stop(self) -> None:
 class ScServer(ScServerAbstract):
     def __init__(self, sc_server_url: str) -> None:
         self._url: str = sc_server_url
-        self._modules: list[ScModuleAbstract] = []
+        self._modules: set[ScModuleAbstract] = set()
```
Comment (AlexandrZagorskiy) [inline src/sc_kpm/sc_server.py:61]: In _src/sc_kpm/sc_module.py_ you use import and **Set** type, but here you use casual **set** type. I think better to unify it

File: src/sc_kpm/sc_server.py [hunk 2]
```diff
@@ -73,14 +77,21 @@ def disconnect(self) -> None:
     def add_modules(self, *modules: ScModuleAbstract) -> None:
         if self.is_registered:
             self._register(*modules)
-        self._modules.extend(modules)
+        self._modules |= {*modules}
```
Comment (AlexandrZagorskiy) [inline src/sc_kpm/sc_server.py:80]: Did we have tests for **ScServer** class? I think this operator was included in python 3.10+ versions, so library wouldn't be compatible with previous versions of Python

**Review by AlexandrZagorskiy — CHANGES_REQUESTED** [review]

File: src/sc_kpm/sc_server.py (hunk 2 above)
Comment (FalseR20) [reply src/sc_kpm/sc_server.py:80]: That's set's operator, it was defined long time ago. Also I write on python version 3.8, so there will be no problems with it

File: src/sc_kpm/sc_server.py (hunk 1 above)
Comment (FalseR20) [reply src/sc_kpm/sc_server.py:61]: In this file we use `__future__.annotations`

**Review by AlexandrZagorskiy — APPROVED** [review]

**Pull request merged**